### ISSUE TITLE
Linux wheels must be named "manylinux1" instead of "linux".

### ISSUE THREAD

**Issue by robertnishihara** [ray-project/ray#624]
When attempting to upload linux wheels to pypi, I ran into the following error.

```
HTTPError: 400 Client Error: Binary wheel for an unsupported platform for url: https://testpypi.python.org/pypi
```

Renaming files as follows `ray-0.1.0-cp36-cp36m-linux_x86_64.whl` -> `ray-0.1.0-cp36-cp36m-manylinux1_x86_64.whl` seemed to fix the problem.

cc @rshin

**Comment by rshin** [comment]
auditwheel does the renaming for you, and also checks that the wheel is
compliant. However, I found that auditwheel would complain about not
meeting the specification, although that may have changed now that the
binaries are being copied differently. I don't know if PyPI checks that the
wheels are indeed manylinux1 compliant.

**Comment by robertnishihara** [comment]
Oh that's good to know. And yes, when I run `auditwheel` it fails with

```
RuntimeError: Invalid binary wheel, found shared library "libray_redis_module.so" in purelib folder.
```

**Comment by robertnishihara** [comment]
I just tried running `auditwheel show` on one of the TensorFlow wheels `tensorflow-1.2.0rc1-cp27-cp27mu-manylinux1_x86_64.whl`, and it gave essentially the same error.

```
RuntimeError: Invalid binary wheel, found shared library "_pywrap_tensorflow_internal.so" in purelib folder.
The wheel has to be platlib compliant in order to be repaired by auditwheel.
```

So I wonder how common it is to make this work.

**Comment by rshin** [comment]
Yes, I noticed they were broken too. But I think the Arrow wheels were compliant :)
I think it's probably not worthwhile to spend more time into this unless someone has problems.

**Comment by robertnishihara** [comment]
Sounds good.

**Comment by robertnishihara** [comment]
Closing this because it is temporarily fixed by #629.
